google-cloud-logging/src/main/java/com/google/cloud/logging/spi/v2/ConfigServiceV2Client.java

```diff
@@ -543,8 +546,10 @@ public final LogSink updateSink(UpdateSinkRequest request) {
    * </code></pre>
    *
    * @param sinkName Required. The resource name of the sink to delete, including the parent
-   *     resource and the sink identifier. Example: `"projects/my-project-id/sinks/my-sink-id"`. It
-   *     is an error if the sink does not exist.
+   *     resource and the sink identifier:
+   *     <p>"projects/[PROJECT_ID]/sinks/[SINK_ID]"
+   *     "organizations/[ORGANIZATION_ID]/sinks/[SINK_ID]"
```

Comment:
We need to stop validating the sinkName input in deleteSink if it can accept multiple patterns, or we need to start using a resource name oneof